src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2475,6 +2477,28 @@
     crafted: true
 """
 
+helps['resource patch'] = """
+type: command
+short-summary: Update a resource by PATCH request.
+long-summary: It is recommended to use 'az resource update' to update a resource.
+              If the update operation fails, try run this command instead.
+examples:
+  - name: Update a webapp by using the latest api-version whether this version is a preview version.
+    text: >
+        az resource patch --ids /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Web/sites/{WebApp} \\
+          --latest-include-preview
```

Comment:
```suggestion
```
In this example, no properties have been updated, could we remove it?

Reply:
To introduce the usage of `--latest-include-preview`, I think it it better to remain this example. I will add updated parameter for this example.